README.md

```diff
@@ -18,6 +18,104 @@ Whether you're starting a new project or refactoring an existing one, you should
 The only end user functionality provided out of the box is basic support for creating organisations and users. The 
 sample code demonstrates end-to-end command handling and event processing flows from API endpoints down to projections.  
 
+
+# Table of Contents
+- [Tooling](#tooling)
+    - [IntelliJ configuration](#intellij-configuration)
+	- [Building](#building)
+	- [Semantic versioning](#semantic-versioning)
+	- [Jupyter notebook](#jupyter-notebook)
+	- [Swagger documentation](#swagger-documentation)
+	- [Checkstyle and PMD configuration](#checkstyle-and-pmd-configuration)
+- [Features](#features)
+    - [Axon: DDD and event sourcing](#axon-ddd-and-event-sourcing)
+    - [Distributed command handling](#distributed-command-handling)
+    - [Command validation](#command-validation)
+    - [Event processing](#event-processing)
+    - [Event replays](#event-replays)
+    - [Security support](#security-support)
+        - [User and Authentication Context](#user-and-authentication-context)
+        - [Endpoint access control](#endpoint-access-control)
+    - [File support](#file-support)
+        - [Configuring Mongo GridFS](#configuring-mongo-gridfs)
+        - [Configuring AWS S3](#configuring-aws-s3)
+    - [Media support](#media-support)
+    - [ETag HTTP headers](#etag-http-headers)    
+- [Project Info](#project-info)
+    - [Maintainers](#maintainers)
+    - [Contributing](#contributing)
+    - [License](#license)
+
+
+# Tooling
+This project uses [Java 11](https://openjdk.java.net/projects/jdk/11/).
+
+The container system is [Docker](https://www.docker.com/).
+
+[Project Lombok](https://projectlombok.org/) greatly reduces the need for hand cranking tedious boilerplate code.
+
+The build system is [Gradle](https://gradle.org/) and it supports all of the above.
+
+## IntelliJ configuration
```

Comment:
Should this title be a more general IDE configuration instead of IntelliJ specific?